ui/tests/acceptance/features/accounts.feature

```diff
@@ -1,8 +1,7 @@
 Feature: Accounts
 
 	Scenario: list accounts
-		Given user "user1" has been created with default attributes
-		And user "user1" has logged in using the webUI
+		Given user "058bff95-6708-4fe5-91e4-9ea3d377588b" has logged in using the webUI
```

Comment:
You should login with username `Moss` here. It takes the username. The UUID doesn't work here.

Reply:
Thanks for the hint! :-)
I was seeing glitches where the accounts service was trying to find the user by the filename `Moss` on disk instead of the account uuid. But I guess that was because of the `created` step in the feature files, which is not needed. Changed it back, and still excluded the `created` step. Let's see what CI says.